### PR TITLE
vendor: update containerd and dependencies to ed261720c86d1e700cd5d39175128322baac6dda

### PR DESCRIPTION
**Pull request by thaJeztah**

### CODE REVIEW

File: vendor.conf [hunk 1]
```diff
@@ -27,23 +27,24 @@ github.com/gogo/protobuf                            5628607bb4c51c3157aacc3a50f0
 github.com/golang/protobuf                          d23c5127dc24889085f8ccea5c9d560a57a879d8 # v1.3.3
 github.com/google/uuid                              0cd6bf5da1e1c83f8b45653022c74f71af0538a4 # v1.1.1
 github.com/grpc-ecosystem/go-grpc-prometheus        c225b8c3b01faf2899099b768856a9e916e5087b # v1.2.0
+github.com/hashicorp/go-multierror                  886a7fbe3eb1c874d46f623bfa70af45f425b3d1 # v1.0.0
```
Comment (thaJeztah) [inline vendor.conf:30]: New dependency; https://github.com/containerd/containerd/commit/0caa2331581ec9e2bc489ceb33fa7280fe6eacad (https://github.com/containerd/containerd/pull/4162)

Comment (dims) [reply vendor.conf:31]: Looks like go-multierror is MPL 2.0 and we already have dependencies using the same license

+1

**Comment by dims** [comment]
LGTM

**Pull request merged**